CSharp/BatchExplorer/Views/CreateControls/AddTaskControl.xaml.cs

```diff
@@ -19,5 +21,18 @@ public AddTaskControl(AddTaskViewModel viewModel)
             this.viewModel = viewModel;
             this.DataContext = viewModel;
         }
+
+        private void ButtonSpinner_Spin(object sender, Xceed.Wpf.Toolkit.SpinEventArgs e)
+        {
+            ButtonSpinner spinner = (ButtonSpinner)sender;
+            TextBox txtBox = (TextBox)spinner.Content;
+
+            int value = String.IsNullOrEmpty(txtBox.Text) ? 0 : Convert.ToInt32(txtBox.Text);
```

Comment:
Could we use an IntegerUpDown instead of a ButtonSpinner with this code-behind updating of the value?  (I'm not familiar with the Xceed toolkit.)